docs/source/examples/ddd_tutorial.myst

```diff
@@ -169,28 +167,25 @@ noise_scaled_expectation_values = []
 for depth in depths:
     print("Status: On depth", depth, end="\n\n")
 
-    true_depth_values, noisy_depth_values = [], []
-    ddd_depth_values = []
-
-    noise_scaled_expectation_depth_values = []
+    true_depth_values, noisy_depth_values, ddd_depth_values = [], [], []
 
     for trial in range(trials):
-        # Local seed is calculated in this way to ensure that we don't get repeat values in loop.
+        # Local seed is calculated in this way to ensure that we don't get repeated values in loop.
         local_seed = 10**6 * depth + 10**3 * seed + trial
 
         circuit, correct_bitstring = get_circuit(depth, local_seed)
 
         true_value = execute(circuit, shots, correct_bitstring, is_noisy=False)
         noisy_value = execute(circuit, shots, correct_bitstring, is_noisy=True)
 
-        ddd_executor = functools.partial(
+        noisy_executor = functools.partial(
             execute,
             shots=shots,
             correct_bitstring=correct_bitstring,
         )
 
         ddd_value = ddd.execute_with_ddd(
-            circuit, ddd_executor, rule=rule, num_trials = 10,
```

Comment:
Note that `num_trials > 1` is typically only useful for **random** DDD sequences. Sence the YY rule is deterministic, there is no need to use multiple trials.